src/libraries/Common/tests/System/Collections/DebugView.Tests.cs

```diff
@@ -11,14 +12,87 @@ namespace System.Collections.Tests
 {
     public class DebugView_Tests
     {
-        public static IEnumerable<object[]> TestDebuggerAttributes_Inputs()
+        private static IEnumerable<object[]> TestDebuggerAttributes_GenericDictionaries()
```

Comment:
Curious, what necessitated the splitting of the test inputs into separate methods?

Reply:
Previously the list contained only generic dictionaries and lists, and all of them were displayed and tested the same way. I added non-generic dictionaries which used a different method of testing.  Depending on the .Net version (Framwork or Core) generic dictionaries now behave similar to elements of either the first or the second group. That's why now there are three groups with two testing methods.